frontend/templates/views/2021/partials/content/link.j2

```diff
@@ -0,0 +1,13 @@
+<a class="ap-link{% if square %} --square{% endif %}"
```

Comment:
Instead of having to manually add the `svg` elements can we instead use an `:before` element with `.--internal` and `.--external` modifier classes?

The thing is I don't want contributors need to care over how icons are used on amp.dev but things to just work.

Reply:
Please have another look. ```.--internal``` is the default. ```.--external```, ```.--square``` and ```.--icon``` are optional modifiers.